Applications/Utils/MeshEdit/AddTopLayer.cpp

```diff
@@ -59,17 +50,23 @@ int main (int argc, char* argv[])
 	cmd.parse(argc, argv);
 
 	INFO("Reading mesh \"%s\" ... ", mesh_arg.getValue().c_str());
-	MeshLib::Mesh * subsfc_mesh(FileIO::readMeshFromFile(mesh_arg.getValue()));
+	auto subsfc_mesh = std::unique_ptr<MeshLib::Mesh>(
+	    FileIO::readMeshFromFile(mesh_arg.getValue()));
```

Comment:
this might be a stupid question but wouldn't it be better to use ```std::unique_ptr<MeshLib::Mesh> subsfc_mesh (...)``` to avoid the new allocation?